README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with example data and NLTK
+Users who wish to avoid using Docker will likely want to use [pip-installed Deephaven](https://deephaven.io/core/docs/tutorials/quickstart-pip/).
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [NLTK](https://nltk.org/).
+TL;DR
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/NLTK/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+pip install deephaven-server deephaven-ipywidgets
 ```
 
-### Launch: Python with example data and PyTorch
-
-Run the following commands to launch Deephaven for Python server applications, with example data and [PyTorch](https://pytorch.org/).
+Then, from Python:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/PyTorch/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```python
+from deephaven_server import Server
+s = Server(port=10000, jvm_args=["-Xmx4g"]).start()
 ```
 
-### Launch: Python with example data and SciKit-Learn
-
-Run the following commands to launch Deephaven for Python server applications, with example data and [SciKit-Learn](https://scikit-learn.org/stable/).
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+### Built from source
 
-### Launch: Python with example data and TensorFlow
+Users who wish to modify source code and contribute to the project should build Deephaven from source. For full instructions, see [how to build Deephaven from source](https://deephaven.io/core/docs/how-to-guides/launch-build/).
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [TensorFlow](https://www.tensorflow.org/).
+TL;DR for Python:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/TensorFlow/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+git clone https://github.com/deephaven/deephaven-core.git
+cd deephaven-core
+python -m venv /tmp/my-dh-venv
+source /tmp/my-dh-venv/bin/activate
+./gradlew py-server:assemble
+pip install "py/server/build/wheel/deephaven_core-<version>-py3-non-any.whl[autocomplete]
+./gradlew server-jetty-app:run
 ```
 
-### Launch: Groovy / Java
-
-Run the following commands to launch Deephaven for Groovy / Java server applications. 
+TL;DR for Groovy:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+git clone https://github.com/deephaven/deephaven-core.git
+cd deephaven-core
+./gradlew server-jetty-app:run -Pgroovy
 ```
 
-### Launch: Groovy / Java with example data
+### Required dependencies
```

Comment:
This section is very awkwardly placed in the doc.  It is after the building instructions.  I suggest moving to the top of the build section, just before the first TLDR.